### PR TITLE
Complete full website

### PR DESCRIPTION
**Pull request by dialite**
This is a calculator project built with react-js and css in this project I did the following:

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Use React Router to make the routes work.
- Use React basic styling.
- Update README.md file .

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Raymond,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby linter checks are not showing on your Pull Request. Kindly ensure that **all** required linters are properly configured with regard to GitHub actions, and **all** checks are passing. Kindly request another code review when this is done.

**Missing Linter Checks:**
image
 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by dialite** [comment]
Thank you for the observation. It has been corrected.

File: src/components/nav.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <ul className="ul-nav">
+        <li className="link-item">
+          <NavLink to="/">
+            Home
+          </NavLink>
+        </li>
+        <li className="link-item">
+          <NavLink to="/calculator">
+            About
+          </NavLink>
```
Comment (Ugboaja-Uchechi) [inline src/components/nav.js:18]: - [ ] Great job 👏. This page should be called `Calculator` not `About` as seen in the project requirements 🙏

File: src/pages/calculator.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/extensions
+import calculate from '../Logic/calculate.js';
```
Comment (Ugboaja-Uchechi) [inline src/pages/calculator.js:3]: - [ ] There's an error while running your application due to the wrong file being imported. It should be `'../logic/calculate'`. Kindly fix this 🙏 

image


- [ ] [OPTIONAL] When importing a `js` file there's no need to add a `.js` extension as it imports all files as `js` by default. Kindly fix this 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @dialite, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/calculator.js (hunk 2 above)
Comment (dialite) [reply src/pages/calculator.js]: Thank you

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hello @dialite 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
# Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/pages/calculator.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/extensions
+import calculate from '../Logic/calculate';
```
Comment (andersonlebon) [inline src/pages/calculator.js:3]: - [ ] As the previous reviewer said, there's an error while running your application due to the wrong file being imported.  Change `'../Logic/calculate'` with `'../logic/calculate'` means '../`L`ogic/calculate' with '../`l`ogic/calculate' => the capital letter `L` with small letter `l`. Kindly fix this :smiley: 
Screenshot from 2022-12-08 21-27-50

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @dialite 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**